packages/website/ts/components/dialogs/wrapped_eth_section_notice_dialog.tsx

```diff
@@ -0,0 +1,38 @@
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+import {colors} from 'material-ui/styles';
+import * as React from 'react';
+
+interface WrappedEthSectionNoticeDialogProps {
+    isOpen: boolean;
+    onToggleDialog: () => void;
+}
+
+export function WrappedEthSectionNoticeDialog(props: WrappedEthSectionNoticeDialogProps) {
+    return (
+        <Dialog
+            title="Dedicated Wrapped Ether Section"
+            titleStyle={{fontWeight: 100}}
+            actions={[
+                <FlatButton
+                    key="acknowledgeWrapEthSection"
+                    label="Sounds good"
+                    onTouchTap={props.onToggleDialog.bind(this)}
```

Comment:
do we want to bind `this` here? I can see someone passing a function here and expecting `this` to refer to to a different context

Reply:
Yep, 100% correct. Good catch.